bcgov/HOUS-permit-portal
# Feature/bphh 373/initial front end shrine override

## Description

This allows S3 uploads via shrine and for form io.
There is a custom uploader for form io, it is not complete at the moment (deletions, reloading, etc.), but the backend work is mostly complete.

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-373

## Steps to QA

You should be able to upload a file and hit submit.  You will NOT be able to see the changes on the UI, but you should be able to inspect the bucket and there should be a corresponding SupportingDocument object.

<img width="1272" alt="Screenshot 2024-01-30 at 9 28 07 AM" src="https://github.com/bcgov/HOUS-permit-portal/assets/607956/739bbf7f-e0aa-4285-a847-e8d6ffbc9f60">

<img width="1266" alt="Screenshot 2024-01-30 at 9 28 16 AM" src="https://github.com/bcgov/HOUS-permit-portal/assets/607956/18cde887-9304-4f19-a121-146b934fc020">


## UI Accessibility Checklist

Default formio should be rendering

## General Checklist

- [ x] ✅ Provide tests for your changes where relevant
- [x ] 📝 Use descriptive commit messages
- [x ] 📗 Update any related documentation and include any relevant screenshots
- [ x] 🗂️ Move your relevant story/task to the _Ready for Code Review_ state
